Source/Scene/getUnpackFloatFunction.js

```diff
@@ -0,0 +1,42 @@
+define([
+        '../Core/defined'
+    ], function(
+        defined) {
+    'use strict';
+
+    var SHIFT_RIGHT_8 = 1.0 / 256.0;
+    var SHIFT_RIGHT_16 = 1.0 / 65536.0;
+    var SHIFT_RIGHT_24 = 1.0 / 16777216.0;
+
+    var BIAS = 38.0;
+
+    /**
+     * Gets a glsl function that takes a vec4 representing a float packed to Uint8 RGBA and returns the unpacked float.
+     *
+     * @param {String} [functionName='unpackFloat'] An optional name for the glsl function.
+     * @returns {String} A string containing a glsl function that takes a vec4 and returns a float.
+     * @private
+     * @see Cartesian4#packFloat
+     */
+    function getUnpackFloatFunction(functionName) {
+        if (!defined(functionName)) {
+            functionName = 'unpackFloat';
```

Comment:
Why does this ever need another name and why can't it just be a built-in `czm_` GLSL function?